requirements/requirements_nlp.txt

```diff
@@ -12,6 +12,7 @@ markdown2
 matplotlib>=3.3.2
 megatron_core==0.5.0
 nltk>=3.6.5
+nvidia-ammo~=0.7.0
```

Comment:
this should be an optional dependency, so remove this

Reply:
done